matrix_client/api.py

```diff
@@ -349,3 +349,19 @@ def _send(self, method, path, content=None, query_params={}, headers={}, api_pat
 
     def media_upload(self, content, content_type):
         return _send("PUT","",content=content,headers={"Content-Type":content_type},apipath="/_matrix/media/r0/upload")
+
+    def get_display_name(self,user_id):
```

Comment:
+1 for consistent code style, however trivial :)

> On 24 Apr 2016, at 07:01, Severen Redwood <notifications@github.com> wrote:
> 
> In matrix_client/api.py:
> 
> > @@ -349,3 +349,19 @@ def _send(self, method, path, content=None, query_params={}, headers={}, api_pat
> >  
> >      def media_upload(self, content, content_type):
> >          return _send("PUT","",content=content,headers={"Content-Type":content_type},apipath="/_matrix/media/r0/upload")
> > +
> > +    def get_display_name(self,user_id):
> Nitpicking, but code style isn't consistent here. The rest of the code separates the function parameters with spaces, whereas in your added functions you haven't separated them.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>